src/herbpy/action/rogue.py

```diff
@@ -0,0 +1,354 @@
+    @param wait The amount of time the object will be held up in seconds
+    @param render Render tsr samples during planning
+    """
+    with robot.GetEnv():
+        if manip is None:
+            manip = robot.GetActiveManipulator()
+        preconfig = manip.GetDOFValues()
+        robot.Grasp(obj)
+
+    #Lift the object - write more tsrs
+    lift_tsr = robot.tsrlibrary(obj, 'lift', manip, distance=distance)
+    
+    with prpy.viz.RenderTSRList(lift_tsr, robot.GetEnv(), render=render):
+        robot.PlanToTSR(lift_tsr, execute=True)
+
+    #Wait for 'time'
+    time.sleep(wait)
+
+    #'Unlift' the object, so place it back down
+    unlift_tsr = robot.tsrlibrary(obj, 'lift', manip, distance=-distance)
+    
+    with prpy.viz.RenderTSRList(unlift_tsr, robot.GetEnv(), render=render):
+        robot.PlanToTSR(unlift_tsr, execute=True)
+
+    if release:
+        with robot.GetEnv():
+            robot.Release(obj)
+        manip.hand.OpenHand()
+        manip.PlanToConfiguration(preconfig)
+
+@ActionMethod
+def NodYes(robot):
+    """
+    @param robot The robot being used to nod
+    """
+    import time
+    pause = 0.15
+    inc = 4
+
+    for i in xrange(inc):
+        robot.head.Servo([ 0,  1])
+        time.sleep(pause)
+    for i in xrange((inc*3)):
+        robot.head.Servo([ 0, -1])
+        time.sleep(pause)
+    for i in xrange((inc*2)):
+        robot.head.Servo([ 0,  1])
+        time.sleep(pause)
+
+@ActionMethod
+def NodNo(robot):
+    """
+    @param robot The robot being used to nod
+    """
+    import time
+    pause = 0.15
+    inc = 4
+
+    for i in xrange(inc):
+        robot.head.Servo([ 1, 0])
+        time.sleep(pause)
+    for i in xrange((inc*3)):
+        robot.head.Servo([-1, 0])
+        time.sleep(pause)
+    for i in xrange((inc*2)):
+        robot.head.Servo([ 1, 0])
+        time.sleep(pause)
+
+@ActionMethod
+def HaltHand(robot, manip=None):
+    """
+    @param robot The robot being used for the stopping gesture
+    @param manip The manipulator being used for the stopping gesture
+    """
+
+    if manip == None:
+        manip = robot.GetActiveManipulator()
+
+    if manip.GetName() == 'right':
+        pose = numpy.array([5.03348748, -1.57569674,  1.68788069,
+                            2.06769058, -1.66834313,
+                            1.53679821,  0.21175342])
+        
+        manip.PlanToConfiguration(pose, execute=True)
+        robot.right_hand.MoveHand(f1=0, f2=0, f3=0, spread=3.14)
+    elif manip.GetName() == 'left':
+        pose = numpy.array([ 1.30614268, -1.76      , -1.57063853,
+                             2.07228362,  1.23918377,
+                             1.46215605, -0.12918424])
+
+        manip.PlanToConfiguration(pose, execute=True)
+        robot.left_hand.MoveHand(f1=0, f2=0, f3=0, spread=3.14)
+    else: 
+        raise prpy.exceptions.PrPyException('Stop is only defined \
+                                                for the left and right arm.')
```

Comment:
This error message is going to contain a bunch of extra spaces. Please wrap it using string concatenation:

```python
raise prpy.exceptions.PrPyException('Stop is only defined'
    ' for the left and right arm.')
```
Or, even better, just wrap outside the string:
```python
raise prpy.exceptions.PrPyException(
    'Stop is only defined for the left and right arm.')
```